pytest_fixtures/component/taxonomy.py

```diff
@@ -108,7 +108,17 @@ def function_entitlement_manifest_org(function_org, function_entitlement_manifes
     return function_org
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture
+def upgrade_entitlement_manifest_org(function_org, upgrade_entitlement_manifest, target_sat):
+    """A Pytest fixture that creates an organization and uploads an entitlement mode manifest
+    generated with Manifester. This will be used for upgrade scenarios"""
+    entitlement_manifest, _ = upgrade_entitlement_manifest
+    function_org.sca_disable()
```

Comment:
Disabling SCA might not be required for 6.11 and 6.12, as it is disabled already there
```suggestion
```